pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -20,9 +22,64 @@ in mkYarnPackage rec {
   inherit version src;
 
   packageJSON = ./element-desktop-package.json;
+  yarnLock = ./element-desktop-yarn.lock;
   yarnNix = ./element-desktop-yarndeps.nix;
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [
+    makeWrapper
+    python3 rustc cargo rustPlatform.cargoSetupHook pkg-config yarn
+    sqlcipher libsecret # FIXME this should be in buildInputs
```

Comment:
nit: which dependency is this comment referring to?

Reply:
sqlcipher and libsecret

Reply:
What should I do about it? Theoretically they belong in buildInputs, but adding them to buildInputs breaks the build.